src/anm/render.js

```diff
@@ -89,8 +89,8 @@ function r_at(time, dt, ctx, anim, width, height, zoom, rib_color, before, after
         } finally { ctx.restore(); }
     } else {
         r_with_ribbons(ctx, width, height,
-                              anim.width, anim.height,
-                              rib_color,
+                            anim.width, anim.height,
+                            rib_color,
```

Comment:
I like grouping things that visually group together, it reads better and feels better than having long lines of arguments.